css-sizing-4/Overview.bs

```diff
@@ -628,6 +618,10 @@ Last Remembered Size</h4>
 	offering the performance benefits of [=size containment=]
 	while <em>probably</em> sizing accurately to its contents.
 
+	Note: ''contain-intrinsic-size: auto'' use in this text is a shorthand representation of
```

Comment:
Prefer deleting this note and just changing  "The ''contain-intrinsic-size: auto'' value" above to instead say "The ''contain-intrinsic-size/auto'' keyword of 'contain-intrinsic-size'"